README.md

```diff
@@ -2,39 +2,59 @@
 
 Here you will find all the [specification types](https://github.com/serverlessworkflow/specification/blob/master/schema/workflow.json) defined by our Json Schemas, in Go.
 
-Some types defined by the specification can be generic objects (such as [`Extensions`](https://github.com/serverlessworkflow/specification/blob/master/extending/README.md)) 
-or share a minimum interface, like [`States`](https://github.com/serverlessworkflow/specification/tree/master/README.md#State-Definition). 
+The final goal of the Serverless Workflow Specification is to offer SDKs that
+have similar features, so users would have the same experience no matter the chosen language.
+Go SDK is a _work in progress_, but should catch up with [Java SDK](https://github.com/serverlessworkflow/sdk-java) soon. 
```

Comment:
I would omit this. Who knows tomorrow Go sdk will have 100x more features and be more useful. I think each sdk can set its own goals and feature sets :)